Assets/MixedRealityToolkit.Services/InputSystem/GazePointerVisibilityStateMachine.cs

```diff
@@ -63,9 +63,8 @@ public void UpdateState(int numNearPointersActive, int numFarPointersActive, int
 
         private void UpdateStateEyeGaze(int numNearPointersActive, int numFarPointersActive)
         {
-            // If there are any far pointers active while eye gaze is valid, then
-            // eye gaze should be disabled.
-            bool isEyeGazePointerActive = numFarPointersActive == 0;
+            // Only enable eye gaze as a pointer if there are no other near or far pointers active.
+            bool isEyeGazePointerActive = numFarPointersActive == 0 && numNearPointersActive == 0;
```

Comment:
It looks like one side effect of this would be that we can no longer use the eye gaze pointer in combination with near interaction, which I think I remember was a use case some of our partners wanted / were using.

Reply:
In that case, they can use EyeTrackingTarget to support simultaneous use of eye gaze and hand input. https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/EyeTracking/EyeTracking_TargetSelection.html#2-independent-eye-gaze-specific-eyetrackingtarget

Reply:
Alternatively we could simply hide the cursor, but then buttons would still be highlighted based on eye gaze, which may be confusing with hands highlighting other buttons.

Reply:
Right... it would be better to make this optional. In the Periodic Table app too. The user can grab and manipulate the element while reading through the text panel with ET auto-scroll.

![2019-08-13 11_42_55-Photos](https://user-images.githubusercontent.com/13754172/62968251-820ffd80-bdbf-11e9-83c7-b9ab63a3fd58.png)

Reply:
ET auto scroll should already handle this, as this was supposed to work even with hand rays active. I might be wrong. Worth testing it again.

Reply:
I see. I'll also test with the Periodic Table app

Reply:
@sostel @keveleigh Verified. Eye-tracking scroll works with while manipulating objects with hands.

Reply:
Yay! :)